shell/browser/api/atom_api_system_preferences_mac.mm

```diff
@@ -503,19 +504,25 @@ AVMediaType ParseMediaType(const std::string& media_type) {
   return AXIsProcessTrustedWithOptions((CFDictionaryRef)options);
 }
 
-std::string SystemPreferences::GetColor(const std::string& color,
-                                        mate::Arguments* args) {
+std::string SystemPreferences::GetColor(gin_helper::ErrorThrower thrower,
+                                        const std::string& color) {
   NSColor* sysColor = nil;
   if (color == "alternate-selected-control-text") {
     sysColor = [NSColor alternateSelectedControlTextColor];
+    EmitDeprecationWarning(node::Environment::GetCurrent(thrower.isolate()),
+                           "Use selected-content-background instead.",
+                           "electron");
   } else if (color == "control-background") {
     sysColor = [NSColor controlBackgroundColor];
   } else if (color == "control") {
     sysColor = [NSColor controlColor];
   } else if (color == "control-text") {
     sysColor = [NSColor controlTextColor];
-  } else if (color == "disabled-control") {
+  } else if (color == "disabled-control-text") {
     sysColor = [NSColor disabledControlTextColor];
+    EmitDeprecationWarning(node::Environment::GetCurrent(thrower.isolate()),
+                           "Use a color that matches the semantics being used.",
```

Comment:
Not sure what this is for, if I don't understand the warning users won't either 😄

Reply:
Looks like it's not deprecated anymore? i'm very confused but rolling with it. Removed.